cis194/week3/daniel-deng/src/Exercise03.hs

```diff
@@ -0,0 +1,24 @@
+module CIS194.Homework03.Exercise03 (histogram) where
+
+import Data.List
+
+stars :: Int -> String
+stars = flip replicate '*'
+
+spaces :: Int -> String
+spaces = flip replicate ' '
+
+starLine :: Int -> String
+starLine nTimes = spaces (10 - nTimes) ++ stars nTimes
+
+count :: Eq a => a -> [a] -> Int
+count x = length . filter (==x)
```

Comment:
@shineli1984 I find it hard to believe that there's [_nothing in hoogle_](https://hoogle.haskell.org/?hoogle=Eq%20a%20%3D%3E%20a%20-%3E%20%5Ba%5D%20-%3E%20Int) for such a trivial function?

Reply:
is this the function? https://hackage.haskell.org/package/MissingH-1.4.2.1/docs/Data-List-Utils.html#v:countElem

Reply:
Yep I noticed that one, not in haskell platform though. Is `MissingH` a widely used package in production code?